examples/source_examples/sql_server/configuration.json

```diff
@@ -0,0 +1,7 @@
+{
+  "driver": "YOUR_ODBC_DRIVER_LIKE='{ODBC Driver 18 for SQL Server}'",
+  "server": "YOUR_SERVER_ADDRESS_LIKE='tcp:sql_test.database.windows.net,1433'",
```

Comment:
Should we remove this and just mention
<YOUR_ODBC_DRIVER>
<YOUR_SERVER>

Reply:
I wanted to provide an example so that customers can use this as a reference.